src/applications/vre/25-8832/config/form.js

```diff
@@ -0,0 +1,116 @@
+import fullSchema from 'vets-json-schema/dist/28-8832-schema.json';
+import environment from 'platform/utilities/environment';
+import { VA_FORM_IDS } from 'platform/forms/constants';
+import { externalServices } from 'platform/monitoring/DowntimeNotification';
+
+import IntroductionPage from '../containers/IntroductionPage';
+import ConfirmationPage from '../containers/ConfirmationPage';
+import { statusSelection } from './chapters/status-selection';
+import { veteranInformation } from './chapters/veteran-information';
+import GetFormHelp from '../../components/GetFormHelp';
+import PreSubmitInfo from '../components/PreSubmitInfo';
+
+import {
+  claimantInformation,
+  claimantAddress,
+} from './chapters/claimant-information';
+import { isDependent, transform } from './helpers';
+import manifest from '../manifest.json';
+
+import { WIZARD_STATUS } from '../constants';
+
+const formConfig = {
+  rootUrl: manifest.rootUrl,
+  urlPrefix: '/',
+  submitUrl: `${environment.API_URL}/v0/education_career_counseling_claims`,
+  trackingPrefix: 'careers-employment-28-8832--',
+  introduction: IntroductionPage,
+  confirmation: ConfirmationPage,
+  transformForSubmit: transform,
+  formId: VA_FORM_IDS.FORM_28_8832,
```

Comment:
`VA_FORM_IDS` should also be updated

Reply:
Agreed this should be updated. I will have a follow up PR that removes the old app (28-8832) once the redirect is in place. Since updating the form id requires approval from other teams, I will just include it in that PR.